### DISCUSSION TITLE
Older version license

### DISCUSSION THREAD

**Discussion by b3d123** [geodesicsolutions-community/geocore-community#192] [Q&A] [answered]
Hi. I'm trying to get one of my older versions working on a domain that it wasn't registered for. How do I get by the License Key? Thanks for any help.

**Comment by vicos59** [comment, marked as answer]
If you're running v18.02.0, you can install the license removal patch here:

https://geodesicsolutions.org/resources.php

If you're running an older version, you'll probably have to upgrade to v20.0.0

**Comment by mtbluedog** [comment]
Hi,
The link to the v18.02.0 patch on https://geodesicsolutions.org/resources.php now 404's.
Is there anywhere to get this patch?
Thanks!

**Reply by vicos59** [reply]
Looks like the patch didn't get moved to the new server. I'll have to look to see if I have a copy.

@blufyremedia @jonyo Do one of you guys have the patch file?

**Comment by vicos59** [comment]
[unencoded_unlicense_patch_18.02.0.bff354dbba748dd30c1d9a7d1ccc7dd61a1b95fc.zip](https://github.com/geodesicsolutions-community/geocore-community/files/13883596/unencoded_unlicense_patch_18.02.0.bff354dbba748dd30c1d9a7d1ccc7dd61a1b95fc.zip)

This looks like it...has the same filename as the broken link. @blufyremedia can you upload this to the web server?

**Comment by blufyremedia** [comment]
Hello @mtbluedog  The issue with the v18.02 patch download on the community website has been resolved.

Otherwise, Vicos looks to have already got you sorted with the files themselves.

**Reply by mtbluedog** [reply]
Thanks! I got the patch. One question: I'm at 18.02.0 and don't have an upgrades folder. So does that mean this was an original 18.02.0 install? And I don't need `updateFactory.unenc.php`?

**Reply by blufyremedia** [reply]
Correct, that folder and file is only if you are running an upgrade from an earlier version to a new one and should be removed after the upgrade is complete. It doesn't indicate whether it was a fresh install, or previously upgraded.

**Reply by mtbluedog** [reply]
Thanks! That's a big help.
